### ISSUE TITLE
Net users output doesnt display correctly

### ISSUE THREAD

**Issue by iNoSec2** [HavocFramework/Havoc#115]
Like we talk in this issue https://github.com/HavocFramework/Havoc/issues/101 and even after your update and my git pull, the output is still broken.
![image](https://user-images.githubusercontent.com/51358868/193442928-9f4276c1-8e09-42d4-b6cb-b0e99e8bba40.png)

**Comment by Cracked5pider** [comment]
it doesn't seem like you git pulled. can you show the source Teamserver/pkg/agent/demons.go line 2975 (and the next 20 lines) ?

**Comment by iNoSec2** [comment]
![image](https://user-images.githubusercontent.com/51358868/193449011-013ab1f9-54ef-4452-89e3-f3e8ae340800.png)

![image](https://user-images.githubusercontent.com/51358868/193449028-31d6324a-e939-44a7-81b2-37390a525ec0.png)

**Comment by Cracked5pider** [comment]
Can you delete the havoc files and git clone again ?

**Comment by Cracked5pider** [comment]
Yeah, it seems that I can't reproduce it. do you mind git cloning the entire project and trying it again? 
![image](https://user-images.githubusercontent.com/51360176/194175246-38cd9b45-bd5f-4bd7-bb37-8de314b8b961.png)
